spec/new_relic_spec.rb

```diff
@@ -26,12 +38,30 @@
   end
 
 
+  context 'with correct setup' do
+    include_examples 'loads'
+  end
+
   context 'when NEW_RELIC_LICENSE_KEY is missing' do
     before do
       app_helper.comment_lines app_path.join('.env'), /NEW_RELIC_LICENSE_KEY/
     end
   
-    include_examples 'abort early', /NEW_RELIC_LICENSE_KEY is required/
+    context 'in the test environment' do
+      let(:app_env) { 'test' }
+
+      include_examples 'does not load'
```

Comment:
So, I checked again, and when New Relic gets loaded in `test`, this sort of thing happens to your test suite (from `logistics-dashboard`):

```
.** [NewRelic][2017-02-08 17:41:09 +0100 salsa.local (35030)] ERROR : Error during start callback for event 'process_action.action_controller':
** [NewRelic][2017-02-08 17:41:09 +0100 salsa.local (35030)] ERROR : NameError: uninitialized constant AnonymousController
** [NewRelic][2017-02-08 17:41:09 +0100 salsa.local (35030)] ERROR : Debugging backtrace:
....rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/newrelic_rpm-3.17.2.327/lib/new_relic/agent/instrumentation/action_controller_subscriber.rb:78:in `const_get'
.../logistics-dashboard/spec/controllers/concerns/oauth_session_required_spec.rb:14:in `block (2 levels) in <top (required)>'
```

ie. the agent gets really confused with the anonymous/dynamic stuff going on in a test suite.